helm/cron_jobs/replayer-cronjobs/templates/cronjobs.yaml

```diff
@@ -0,0 +1,60 @@
+{{- $rootContext := . }}
+{{- range $replayer := .Values.replayers }}
+{{- with $replayer -}}
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "replayer-cronjobs.fullname" $rootContext }}-{{ .name }}
+spec:
+  successfulJobsHistoryLimit: 3
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 1
+  schedule: {{ .schedule }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - command:
+            - /bin/bash
+            - -c
+            - /scripts/{{ .script.scriptName }}
+            env:
+            - name: GCLOUD_KEYFILE
+              value: /gcloud/keyfile.json
+            name: {{ .name }}
+            image: {{ .image.repository }}:{{ .image.tag }}
+            imagePullPolicy: {{ .image.pullPolicy }}
+            {{- with .resources }}
+            resources:
+            {{- toYaml . | nindent 14 }}
+            {{- end }}
+            terminationMessagePath: /dev/termination-log
+            terminationMessagePolicy: File
+            volumeMounts:
+            - mountPath: /gcloud/
+              name: gcloud-keyfile
+            - mountPath: /scripts/
+              name: {{ .script.secretName }}
+          restartPolicy: Never
+          terminationGracePeriodSeconds: 30
+          volumes:
+          - name: gcloud-keyfile
+            secret:
+              defaultMode: 256
+              secretName: gcloud-keyfile
```

Comment:
If we are changing the name of the secret (as mentioned [here](https://github.com/MinaProtocol/mina/pull/14942/files#r1462963402)), then this should match